bin/RUV_normalisation_gtex.R

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env Rscript
+
+# Author: Suhaib Mohammed
+
+ ## parse study_list as `echo -e "E-MTAB-5214"'\t'"E-MTAB-513"'\t'"E-MTAB-2836"`
+
+suppressMessages( library( ExpressionAtlas) )
+suppressMessages( library( funr) )
+
+script_path<-funr::get_script_path()
+
+## loading generic functions 
+source(paste(script_path,"generic_functions.R",sep="/"))
+
+# Get commandline arguments.
+args <- commandArgs( TRUE )
+
+if( length( args ) != 2 ) {
+  stop( "\nUsage:\n\tscript.R <study_list> <output_path>\n\n" )
+}
+
+# Get the transcripts expression filename from the arguments.
+study_list <- args[ 1 ]
+output_path <- args[ 2 ]
+
+study_list <- unlist(strsplit(study_list, split="\t"))
```

Comment:
Using tabs here is probably error-prone, I would suggest a more distinctive separator for cli purposes (such as `,` or `;`.